M1FExplained/Chapter25/Exercises/Exercise04.lean

```diff
@@ -0,0 +1,33 @@
+import Mathlib.Algebra.Group.Defs
+import Mathlib.Algebra.GroupPower.Basic
+import Mathlib.Data.Real.Basic
+import Mathlib.Logic.Equiv.Defs
+import Mathlib.Logic.Equiv.Basic
+import Mathlib.Data.Finset.Basic
+import Mathlib.GroupTheory.Perm.Basic
+
+/- a) Show that if G is an abelian group and n is an integer, then (a * b)^n = a^n * b^n for all a, b, ∈ G.
+   b) Give an example of a group G, an integer n, and elements a, b ∈ G such that (a * b)^n ≠ a^n * b^n. -/
+
+
+--- part a)
+
+example {G : Type u_1} [CommGroup G] (a b : G) (n : ℕ) : (a*b)^n = a^n*b^n := by
+  induction' n with n ih
+  · rw [pow_zero, pow_zero, pow_zero, mul_one]
```

Comment:
The simplifier will know `pow_zero` and `mul_one` because both of them are manifestly great simp lemmas: the LHS is obviously more complex than the RHS and always wants to be simplified to it. So probably `simp` works fine here (as it often does in the base case of an induction).